task6/script.js

```diff
@@ -0,0 +1,333 @@
+            likes: ["ar_kud"]
+        },
+        {
+            id: '14',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2020-05-03T18:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '15',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2019-05-21T17:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '16',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2020-05-03T13:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '17',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2020-05-03T17:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '18',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2020-05-03T17:31:30'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '19',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2019-06-03T17:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        },
+        {
+            id: '20',
+            description: "Как же хорошо, что у меня наконец появилось время подтянуть все свои долги!",
+            createdAt: new Date('2020-05-01T17:31:24'),
+            author: "ar_kud",
+            name: "Artsiom Kudryavtsev",
+            photoLink: "zZuTNe0PW34.jpg",
+            hashtags: [],
+            likes: ["ar_kud"]
+        }
+    ];
+    var deletedPosts = [];
+
+    function getPosts(skip, top, filterConfig) {
+        skip = skip || 0;
+        top = top || 10;
+
+        var result = posts.slice();
+
+        if (filterConfig.hashtags !== undefined && filterConfig.hashtags !== null) {
```

Comment:
`if (filterConfig.hashtags)` will be enough (in other places too)

Reply:
Done.